code/modules/admin/admin_verbs.dm

```diff
@@ -633,9 +633,10 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 
 	holder.deactivate()
 
-	to_chat(src, "<span class='interface'>You are now a normal player.</span>", confidential = TRUE)
-	log_admin("[src] deadmined themself.")
-	message_admins("[src] deadmined themself.")
+
+	to_chat(src, "<span class='interface'>You are now a normal player.</span>")
+	log_admin("[src] deadminned themselves.")
```

Comment:
Why change it to a plural pronoun when it's referring to one person?

Reply:
'cause there's no gender-neutral singular pronoun. It's them or nothing.